config.yaml

```diff
@@ -33,10 +33,9 @@ imaging:
 
 languages:
   en:
-    title: Notary | Signing and verifying artifacts. Safeguarding the software delivery security from development to deployment.
+    title: Notary Project | A set of specifications and tools intended to provide a cross-industry standard for securing software supply chains.
     description: >-
-      The Notary project comprises a server and a client for running and
-      interacting with trusted collections.
+      The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains by using authentic container images and other OCI artifacts.
```

Comment:
```suggestion
The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains by using authentic container images and other OCI artifacts. Notary Project is also the name of the GitHub organization that has multiple prominent subprojects like Notation, Notary Project specifications, and Notary. Very often we use the name Notary Project to refer to all the above as well as the community that drives the specifications and the implementations.
```

Reply:
This is the website metadata that is used for SEO purposes. It should be neat. It's not recommended to replicate too many words from the whole introduction to the website metadata.